.github/workflows/deploy.yml

```diff
@@ -0,0 +1,63 @@
+name: Deploy Netlify
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: subosito/flutter-action@v2.16.0
+        with:
+          flutter-version: '3.22.2'
+
+      - name: make .env
+        working-directory: flutter_bird_app
+        run: |
+          touch .env
+          echo WALLET_CONNECT_PROJECT_ID=${{secrets.WALLET_CONNECT_PROJECT_ID}} >> .env
```

Comment:
I am currently using String.fromEnvironment, 
https://github.com/Finschia/flutter-bird/pull/17/files#diff-ce468568a22cb6da6b1602afea679053f64152784d721976839f5baa4977b352R44

but according to this can both dev and prod be managed in .env?

Reply:
Yes, if you want to separate dev and prod, you can do so as shown below.
https://github.com/Finschia/flutter-bird/pull/18/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R45-R49